lithops/standalone/standalone.py

```diff
@@ -183,13 +184,18 @@ def invoke(self, job_payload):
 
         def create_workers(workers_to_create):
             current_workers_old = set(self.backend.workers)
+            futures = []
             with cf.ThreadPoolExecutor(workers_to_create+1) as ex:
-                ex.submit(lambda: self.backend.master.create(check_if_exists=True)
-                          if not self._is_master_service_ready() else False)
+                if not self._is_master_service_ready():
+                    futures.append(ex.submit(lambda: self.backend.master.create(check_if_exists=True)))
+
                 for vm_n in range(workers_to_create):
                     worker_id = "{:04d}".format(vm_n)
                     name = f'lithops-worker-{executor_id}-{job_id}-{worker_id}'
-                    ex.submit(self.backend.create_worker, name)
+                    futures.append(ex.submit(self.backend.create_worker, name))
```

Comment:
I think this can introduce a potential issue from our perspective: if for some reason one worker cannot be created, this will raise the exception and cancel the whole job. The idea here is to start as much workers as possible, and then proceed to run the job with those workers.

IBM paid account have a limitations of 200vCPUs per region. This means a user can start at most 100VMs (cx2-2x4). In our case we are 6 users using the same IBM account and most of times the same region. And the regions is not fully dedicated to Lithops, we also have other VMs running other services, so the number of vCPUs we can dedicate to Lithops is much lower. In most of the times, this limitation makes impossible to create the desired workers. So if the exceptions are raised here it is possible that this prevents us to run any job.

I think exceptions are already logged in the backends. And in the case that there is another exception none related to Quota Limit, no one VM will be created, and then, few lines below there is already a check for this and it will already raise an exception.

Reply:
Thanks for the explanation, now I understand the use case. In this approach user can be left with a single worker while expecting to have N workers. This may result in unexpected computations results.
My suggestion - to make both scenarios possible. I will add to standalone config `workers_policy` that will have currently 2 options:

workers_policy: permissive     # in this case it will ignore worker create errors on client as in current implementation and will continue to function invocation even with a single worker 
workers_policy: strict              # in this case it will force availability of N workers as user expected

What do you think?

Reply:
Good idea, this makes much more sense

Reply:
Done